sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/recordings/test_query_knowledgebase.pyTestQnAKnowledgeBasetest_query_knowledgebase.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
Why are the test recordings here? Please follow the instructions from here https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md